R/deprecated-prob_helpers.R

```diff
@@ -0,0 +1,68 @@
+# `...` -> estimate matrix / vector helper -------------------------------------
+
+#' Developer helpers
+#'
+#' Helpers to be used alongside [check_metric], [yardstick_remove_missing] and
+#' [metric summarizers][class_metric_summarizer()] when creating new metrics.
+#' See [Custom performance
+#' metrics](https://www.tidymodels.org/learn/develop/metrics/) for more
+#' information.
+#'
+#' @section Dots -> Estimate:
+#' `r lifecycle::badge("deprecated")`
```

Comment:
Are there other deprecated functions on this help page? You might just want to move the deprecation badge into the Description if not

Reply:
The page includes documentation for `dots_to_estimate()`, `get_weights()`, `finalize_estimator()`, `finalize_estimator_internal()`, and `validate_estimator()`

And it looks like this:

![Screen Shot 2023-02-07 at 2 14 49 PM](https://user-images.githubusercontent.com/14034784/217378999-b9429764-04d2-4922-bb5b-e79ca740c5e4.png)

Are you suggesting that I move the deprecation warning all the way up?

Reply:
What you have is fine then!